tests/test_transformations.py

```diff
@@ -476,7 +476,7 @@ class Hm(Mapping):
         second = {}
         third = {"wow": 3}
         result1 = Hm().apply(first).serialize()
-        result2 = Hm().apply(second).serialize()
+        result2 = Hm().apply(second).serialize(implicit_nulls=True)
```

Comment:
could go either way on this, but i think change the assertion instead: `assert result2 == {"wow": None}`